### ISSUE TITLE
Unable to find template

### ISSUE THREAD

**Issue by saviola777** [marttiphpbb/phpbb-ext-customcode#41]
I'm running phpBB 3.2.5 and just installed your extensions via the instructions. After making minor changes to one of the files via the ACP and purging the cache, I get the following error:

```
Fatal error: Uncaught exception 'Twig_Error_Loader' with message 'Unable to find template "../../../../../../store/customcode/overall_header_head_append.html" (looked into: /phpBB/styles/latte/template, /phpBB/styles/latte/theme, /phpBB/styles/prosilver/template, phpBB/styles/prosilver/theme, /phpBB/styles/all/template) in "@marttiphpbb_customcode/event/overall_header_head_append.html" at line 5.'
```

When changing the path in `/phpBB/ext/marttiphpbb/customcode/styles/all/template/event/overall_header_head_append.html` to the more correct looking ´../../../store/customcode/overall_header_head_append.html`, I instead get the error

```
Fatal error: Uncaught exception 'Twig_Error_Loader' with message 'Looks like you try to load a template outside configured directories (../../../store/customcode/overall_header_head_append.html) in "@marttiphpbb_customcode/event/overall_header_head_append.html" at line 5.'
```

What is going on here? Did I make a mistake setting up the extension?

On a more important note, since my forum, including the ACP, is now broken: is there a way to disable or remove the extension manually – will deleting the extension directory and purging the cache restore functionality?

Thanks in advance.

**Comment by eeemarv** [comment]
Do you have the latest version (2.0.2) ?

**Comment by marttiphpbb** [comment]
Do you have the latest version (2.0.2) installed?

**Comment by saviola777** [comment]
I took the latest GitHub release, which is… 2.0.1? I'll try to find the 2.0.2 release then and try it with that… I'll report back.

**Comment by saviola777** [comment]
Okay, what happened is that I checked the released, missed the "show 2 newer tags" button and installed version 1.1.0. I managed to fix my board by disabling all extensions via a separate PHP script. Then I installed 2.0.2 which can't be enabled because my webhoster is sill running PHP5 despite claiming it would disable PHP5 two months ago, I was assuming I was running PHP7.

So yeah, nevermind, sorry for wasting your time. I'll get my board switched to PHP7 and try enabling 2.0.2 again.
